app/controllers/reviews_controller.rb

```diff
@@ -118,4 +119,39 @@ def check_review_period
 
     redirect_to root_path, alert: t('reviews.edit.errors.conference_out_of_range')
   end
+
+  def create_third_reject(review)
+    return if in_early_review_phase?
+
+    return unless review.weak_reject? || review.strong_reject?
+
+    session_reviews = review.session.early_reviews + review.session.final_reviews
+    other_reviews = session_reviews.reject { |r| r == review }
+
+    return if other_reviews.count != 1
+
+    other_review = other_reviews.first
+
+    return unless other_review.weak_reject? || other_review.strong_reject?
+
+    low_rating = Rating.find_by(title: 'rating.low.title')
+    resource_class.create!(
+      session: review.session,
+      recommendation: Recommendation.find_by(name: 'strong_reject'),
+      reviewer_id: 742,
+      author_agile_xp_rating: low_rating,
+      author_proposal_xp_rating: low_rating,
+      proposal_quality_rating: low_rating,
+      proposal_relevance_rating: low_rating,
+      reviewer_confidence_rating: low_rating,
+      proposal_track: false,
+      proposal_level: false,
+      proposal_type: false,
+      proposal_duration: false,
+      proposal_limit: false,
+      proposal_abstract: false,
+      comments_to_authors: '.' * 150,
```

Comment:
This has to be some text explaining the automatic rejection, since the authors will receive it back. I suggest encouraging them to read the other reviewers suggestions attentively and improving the session for next year submissions.